### DISCUSSION TITLE
Report/Render | Download error 404 when I publish project

### DISCUSSION THREAD

**Discussion by timsan7sk** [serenity-is/Serenity#6302] [Help]
Hi for All!
When I debug local(IIS Express) it's work perfect, but when publish on server(IIS8.5)  I have error 404.
web.config
```
<appSettings>
 <add key="EnvironmentSettings" value="{ SiteExternalUrl: 'https://site.url/' }" />
</appSettings>
```
and I have in appsettings.json
```
    "EnvironmentSettings": {
      "SiteExternalUrl": "https://site.url/"
    },
```
I almost broke my head, but I can not understand what is the reason.
Can any body help?

**Comment by minhhungit** [comment]
unlucky I got same issue :(

**Comment by minhhungit** [comment]
Tried to use SiteExternalUrl setting from https://github.com/serenity-is/Serenity/discussions/6343#discussioncomment-2545258
in a sec I thought it works but I was wrong

My problem due to **I got exception on test site**
